index.html

```diff
@@ -1,14 +1,161 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <link rel="stylesheet" href="style.css">
-  <title>Free thinking Conference: The value of reason, logic, and evidence.</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,300;0,400;0,700;0,900;1,300;1,900&display=swap" rel="stylesheet">
+<link rel="stylesheet" href="style.css">
+  <title>Free Thinking Conference: The value of reason, logic, and evidence.</title>
 </head>
-<body>
-  <h1>hello world!</h1>
+
+<body class="body">
+  <div class="mobile-menu hide">
+    <img class="mobile-menu-close" src="images/icons/pop-close.svg" alt="close">
+    <ul class="mobile-menu-elements">
```

Comment:
- [ ] Please use the `nav` tag for your menu/navigation items. This has a lot of advantages when it comes to semantic HTML. Screen readers and SEO need such information to pars your webpage and understand its content.